Gruntfile.js

```diff
@@ -72,16 +72,6 @@ module.exports = function (grunt) {
 				maxBuffer: maxBufferSize
 			},
 
-			coverageUnit: {
```

Comment:
why are these 2 tasks being removed? Corresponding index files in test directories are not needed then and should be removed as well